.dotnet/Azure.AI.OpenAI.Staging/Custom/AzureOpenAIClient.cs

```diff
@@ -0,0 +1,144 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.AI.OpenAI.Staging.Audio;
+using Azure.AI.OpenAI.Staging.Chat;
+using Azure.AI.OpenAI.Staging.Embeddings;
+using Azure.AI.OpenAI.Staging.FineTuning;
+using Azure.AI.OpenAI.Staging.Images;
+using Azure.Core;
+using OpenAI;
+using OpenAI.Assistants;
+using OpenAI.Audio;
+using OpenAI.Chat;
+using OpenAI.Embeddings;
+using OpenAI.Files;
+using OpenAI.FineTuning;
+using OpenAI.Images;
+using OpenAI.ModelManagement;
+using OpenAI.Moderations;
+using System.ClientModel;
+using System.ClientModel.Primitives;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Azure.AI.OpenAI.Staging;
+
+public partial class AzureOpenAIClient : OpenAIClient
+{
+    protected static new internal RequestOptions DefaultRequestOptions => OpenAIClient.DefaultRequestOptions;
+    protected static new internal PipelineMessageClassifier PipelineMessageClassifier200 => OpenAIClient.PipelineMessageClassifier200;
```

Comment:
why is this needed? Why not use the base property?

Reply:
Per Anne's feedback, I've removed the inheritance of these two -- the RequestOptions go away entirely and the classifier becomes per-layer lazy-init property/field pairs.

As the outdated answer, it was just because the `protected internal` access didn't provide visibility to the scenario clients for their protocol method use -- and the main goal was to avoid each and every scenario client having the same thing with initializing these.